ESMF/GOCART2G_GridComp/CA2G_GridComp/CA2G_StateSpecs.rc

```diff
@@ -74,32 +74,32 @@ category: EXPORT
 #----------------------------------------------------------------------------------------
  NAME       | UNITS      | DIMS| VLOC| UNGRIDDED | LONG NAME
 #----------------------------------------------------------------------------------------
- CAMASS*    | kg kg-1    | xyz | C   |       | Carbonaceous Aerosol Mass Mixing Ratio                
- CACONC*    | kg m-3     | xyz | C   |       | Carbonaceous Aerosol Mass Concentration               
- CAEXTCOEF* | m-1        | xyz | C   | size(self%wavelengths_profile) | Carbonaceous Aerosol Extinction Coefficient 
- CASCACOEF* | m-1        | xyz | C   | size(self%wavelengths_profile) | Carbonaceous Aerosol Scattering Coefficient 
+ *MASS    | kg kg-1    | xyz | C   |       | Carbonaceous Aerosol Mass Mixing Ratio                
```

Comment:
After reading #93 I was expecting to see modifiers in the long names as well, e.g: 
    *MASS | kg kg-1 | xyz | C | | ***Carbonaceous Aerosol Mass Mixing Ratio**

Was that feature implemented?